cmd/ctl/pkg/create/certificaterequest/certificaterequest.go

```diff
@@ -192,13 +196,13 @@ func (o *Options) Run(args []string) error {
 		FilenameParam(o.EnforceNamespace, &resource.FilenameOptions{Filenames: []string{o.InputFilename}}).Flatten().Do()
 
 	if err := r.Err(); err != nil {
-		return fmt.Errorf("error when getting Result from Builder: %s", err)
```

Comment:
Any reason we stopped wrapping these errors?

Reply:
Richard mentioned they made it harder to understand the error, and I agreed